analyzer/src/main/kotlin/managers/utils/NuGetSupport.kt

```diff
@@ -109,23 +109,7 @@ class NuGetSupport(
         }
     }
 
-    private val registrationsBaseUrls: List<String> = run {
-        val configFile = definitionFile.parentFile.searchUpwardsForFile("nuget.config", ignoreCase = true)
-        val serviceIndexUrls = configFile?.let { NuGetConfigFileReader.getRegistrationsBaseUrls(it) }
-            ?: listOf(DEFAULT_SERVICE_INDEX_URL)
-        val serviceIndices = runBlocking {
-            serviceIndexUrls.map {
-                async { JSON_MAPPER.readValueFromUrl<ServiceIndex>(it) }
-            }.awaitAll()
-        }
-
-        // Note: Remove a trailing slash as one is always added later to separate from the path, and a double-slash
-        // would break the URL!
-        serviceIndices
-            .flatMap { it.resources }
-            .filter { it.type == REGISTRATIONS_BASE_URL_TYPE }
-            .map { it.id.removeSuffix("/") }
-    }
+    private val registrationsBaseUrls: List<String> = getRegistrationBaseUrls(definitionFile)
```

Comment:
IMO this would have been a fitting change already in the first commit, and could be squashed.

Reply:
I prefer to not squash it, as perfect order is not necessary. This is the order things popped up in the refactoring. IMO it's fine to keep that (mikado method) ordering.